### PR TITLE
improve websocket docs and bump websockets lib version

### PR DESCRIPTION
**Pull request by ledhed2222**
## High Level Overview of Change

Adds more detail to the docs section of the async websocket client to better explain how to use it, especially for people who are coming from JS.

Updates WS library

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

File: xrpl/asyncio/clients/async_websocket_client.py [hunk 1]
```diff
@@ -56,61 +56,152 @@ class AsyncWebsocketClient(AsyncClient, WebsocketBase):
...
+    you can use a Task to listen to messages without blocking. Note this
+    example can be copied directly into a file and run with no changes::
 
         import asyncio
 
         from xrpl.asyncio.clients import AsyncWebsocketClient
         from xrpl.models import Subscribe, Unsubscribe, StreamParameter
 
-        URL = "wss://some-url-to-connect-to"
+        URL = "wss://s.devnet.rippletest.net:51233"
```
Comment (JST5000) [inline xrpl/asyncio/clients/async_websocket_client.py:67]: nit: I'd recommend using testnet instead of devnet here since most people aren't testing new amendments - wss://s.altnet.rippletest.net:51233/

**Review by JST5000 — APPROVED** [review]
LGTM

File: xrpl/asyncio/clients/websocket_base.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 from typing import TYPE_CHECKING, Any, Dict, Optional, cast
 
 from typing_extensions import Final
-from websockets.legacy.client import WebSocketClientProtocol, connect
+from websockets import client as websocket_client
```
Comment (JST5000) [inline xrpl/asyncio/clients/websocket_base.py:10]: Why was this changed in this PR? (Maybe it could be split out?)

**Comment by justinr1234** [comment]
@ledhed2222 would you be able to revisit this PR and update to latest main branch?

File: xrpl/asyncio/clients/websocket_base.py (hunk 2 above)
Comment (ledhed2222) [reply xrpl/asyncio/clients/websocket_base.py:10]: bumped the websocket library while here :)

**Review by JST5000 — APPROVED** [review]
LGTM, sorry for the massive delay!

**Review by intelliot — APPROVED** [review]
if it works, it works
